index.rst

```diff
@@ -98,16 +98,11 @@ public transportation or a taxi there.
 **How should I get around?**
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  Portland is a very walk-able town. Most of the events and `the
-   suggested
-   hotels <http://www.writethedocs.org/conf/na/2017/visiting/>`_ are
-   within walking distance of the conference venue, the Crystal
-   Ballroom.
+Vancouver boasts an accessible and convenient `public transit system <https://www.tourismvancouver.com/plan-your-trip/getting-around/transit-system/>`_, offering a great way for visitors to see the city. Most of Vancouver’s top attractions are reachable via transit. TransLink is Metro Vancouver’s regional transportation authority, and operates a public transit system made up of a network of buses, the SkyTrain, and the SeaBus, as well as the West Coast Express commuter train service.
 
--  There are good public transportation options and taxi services. Check
-   out the `Visiting Portland section of the Write the Docs
-   website <http://www.writethedocs.org/conf/na/2017/visiting/>`_ for
-   more info.
+The `TransLink website <https://www.translink.ca/>`_ is a great place to find current schedules, fare information, and trip planning tools to help you get around while in Vancouver. You can also call 604.953.3333 for customer service.
```

Comment:
Remove this section, or write up your own copy.

Reply:
Done